src/modules/launcher/PowerLauncher/MainWindow.xaml.cs

```diff
@@ -159,7 +160,13 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 
             SearchBox.QueryTextBox.DataContext = _viewModel;
             SearchBox.QueryTextBox.PreviewKeyDown += Launcher_KeyDown;
-            SearchBox.QueryTextBox.TextChanged += QueryTextBox_TextChanged;
+
+            Observable.FromEventPattern<TextChangedEventHandler, TextChangedEventArgs>(
+                add => SearchBox.QueryTextBox.TextChanged += add,
+                remove => SearchBox.QueryTextBox.TextChanged -= remove)
+                    .Throttle(TimeSpan.FromMilliseconds(250))
```

Comment:
Seems a bit to much I think.

We have plugins that return their values in < 1 millisecond and other slow ones need more than 100 milliseconds. Your delay will make showing the results significantl slower and I guess that other users will complaining about that change.

Does this also has impacts on delayed results/plugin queries?

Reply:
@htcfreek. You make good points. I think the intention of the fix mentioned in #11176 is to allow the user to enter a few characters before a search query is performed in hopes the plugins would be able to return faster. But, If plugins return faster than we can detect an input delay, this approach doesn't make sense anyway. No worries if this needs to be closed based on that point.